README.md

```diff
@@ -93,6 +93,22 @@ Diagrams are generated "automatically" by ???
 
 ## User Guide
 
+###Database Management
+To run a local version of this project you will need to start a 
+repl and initialize the vanilla_db.
+This can be done in 
+````$xslt
+src/clj/vanilla/db/core.clj
+````
+In the first comment block.
```

Comment:
maybe we should put a searchable comment  into the code to make this easier to find (and then document that special comment here)

Reply:
I'm not sure what you mean by this. Do you mean something similar to the $xslt but something we create so we can find these code lines?

Reply:
Forget the comment - in the readme be explicit about the function to run in the REPL to generate a new default database (including a link to the file)

Reply:
Done, including making a new md file on the database management. Seems like the way we've been doing it as well.